packages/typespec-vscode/src/utils.ts

```diff
@@ -27,6 +28,19 @@ export async function isDirectory(path: string) {
   }
 }
 
+export async function createTempDir(): Promise<string> {
```

Comment:
add try-catch to avoid unexpected exception from IO which will break the whole extension

Reply:
done